### PR TITLE
EWL-4245 - Remove extraneous bloat code and inline css styles in SVG 

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**

- [EWL-4245: SVG cleanup | SG | Remove inline css and cleanup SVG files](https://issues.ama-assn.org/browse/EWL-4245)

## Description

- Remove bloat code in SVG files using SVGO https://github.com/svg/svgo
- Remove inline css from SVG files

## To Test
- [ ] read readme.md for installation of SVGO
- [ ] Go to http://localhost:3000/?p=viewall-atoms-all to verify icons are not broken
- [ ] Check SVG files to see if inline css has been removed
- [ ] Check SVG files and verify bloat code has been removed

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks

N/A

## Additional Notes
I used [SVGO](https://github.com/svg/svgo) to remove all bloat code in our SVG's. We will have to use it in the future to remove bloat when we receive SVG files.

### CODE REVIEW

**Comment by chelsiejohnston** [comment]
Also want to confirm with @froboy that this change won't affect the way gulpicon works currently

**Comment by designerneil** [comment]
@chelsiejohnston I would normally agree with you on adding a task in gulp to clean up SVGs but think every SVG should be optimized before it's added into the repo by the developer working on it. If this becomes an issue in the future then we can address it by using a gulp task.

**Comment by designerneil** [comment]
@froboy the documentation is in https://github.com/svg/svgo

**Comment by froboy** [comment]
@designerneil we should definitely put something either in `README.md` or `/docs` that says "When you add an SVG, you should _________". Feel free to link to svgo, but we need to call that out in the docs for posterity (and future you).

**Comment by designerneil** [comment]
@froboy thanks. I've updated the readme.md

File: styleguide/README.md [hunk 1]
```diff
@@ -91,3 +91,9 @@ To install a near-empty StarterKit as a starting point for your project type:
 To install a specific StarterKit from GitHub type:
 
     php core/console --starterkit --install <starterkit-vendor/starterkit-name>
+
+
+### Slim and remove bloat from SVG files: install and run SVGO
```
Comment (froboy) [inline styleguide/README.md]: Sorry to keep pushing back, but this should go in the root readme, not the PL-specific one. Also, please add some narrative as I described above. You should work under the assumption that the user reading the documentation doesn't know what SVGO is or when/why they should use it.

**Comment by designerneil** [comment]
@froboy readme updated per your suggestion

**Comment by froboy** [comment]
@designerneil thanks! Looks good.

**Pull request merged**